coderd/database/modelmethods.go

```diff
@@ -121,8 +134,26 @@ func (u User) UserDataRBACObject() rbac.Object {
 	return rbac.ResourceUser.WithID(u.ID).WithOwner(u.ID.String())
 }
 
-func (License) RBACObject() rbac.Object {
-	return rbac.ResourceLicense
+func (u GetUsersRow) RBACObject() rbac.Object {
+	return rbac.ResourceUser.WithID(u.ID)
+}
+
+func (u GitSSHKey) RBACObject() rbac.Object {
+	return rbac.ResourceUserData.WithID(u.UserID).WithOwner(u.UserID.String())
+}
+
+func (u GitAuthLink) RBACObject() rbac.Object {
+	// I assume UserData is ok?
+	return rbac.ResourceUserData.WithID(u.UserID).WithOwner(u.UserID.String())
```

Comment:
this looks like the only actual usage of `ResourceUserData` -- why did we add it in the first place?

Reply:
I did a "find usage" on `ResourceUserData` on `main` and the only reference is tests.  So, I don't think the current implementation actually checks for `ResourceUserData` before doing things like updating user passwords/profiles/GitSSHKeys.

Reply:
This is from `main`. https://github.com/coder/coder/blob/33f2c8fef5fed25209d08896ca20e01d1655ea91/coderd/gitsshkey.go#L38-L41

It was supposed to, but the modelmethod was incorrect.